README.md

```diff
@@ -22,6 +22,20 @@ main() {
 }
 ```
 
+## Cron parser
+You can easily create and parse [cron format](https://www.ibm.com/docs/en/db2oc?topic=task-unix-cron-format)
```

Comment:
I'd rather link to https://en.wikipedia.org/wiki/Cron here, I think it is more neutral.

Reply:
> I'd rather link to https://en.wikipedia.org/wiki/Cron here, I think it is more neutral.

sure